fe3.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <style rel="stylesheet" href="stylesheet/3css.css"></style>
+</head>
+<body>
+  <section class="promo">
+    <h1 class="promo__title">Press</h1>
+    <p class="promo__description">Get weekly articles in your inbox on how to grow your business</p>
+    <form class="promo__form">
+      <input class="field" type="email" value="Enter your email">
+      <input class="button" type="button" value="Notify me">
+    </form>
+  </section>
+  <article>
+    <section class="section">
+      <h3 class="section__title">Cool story</h3>
+      <time class="section__date" datetime="2020-03-16T12:34:56:78900">Mar 16, 2020</time>
+      <p class="section__text">
+        I contend that we are both atheists. I just believe in one fewer god
+        than you do. When you understand why you dismiss all the other possible
+        gods, you will understand why I dismiss yours.
+      </p>
+      <footer class="section__footer"><a class="section__link" href="#">Read more</a></footer>
+    </section>
```

Comment:
предложу поправить на такое,
ну и название блока section, странное

```suggestion
  <div class="promo">
    <h1 class="promo__title">Press</h1>
    <p class="promo__description">Get weekly articles in your inbox on how to grow your business</p>
    <form class="promo__form">
      <input class="field" type="email" value="Enter your email">
      <input class="button" type="button" value="Notify me">
    </form>
  </div>
  <div>
    <article class="section">
      <h3 class="section__title">Cool story</h3>
      <time class="section__date" datetime="2020-03-16T12:34:56:78900">Mar 16, 2020</time>
      <p class="section__text">
        I contend that we are both atheists. I just believe in one fewer god
        than you do. When you understand why you dismiss all the other possible
        gods, you will understand why I dismiss yours.
      </p>
      <footer class="section__footer"><a class="section__link" href="#">Read more</a></footer>
    </article >
    .....
    .....
    .....
    </div>
```